packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -161,6 +161,11 @@ abstract class Route<T> extends _RoutePlaceholder {
     }
   }
 
+  /// When the route state is updated, request focus if the current route is at the top.
+  ///
+  /// Defaults to true.
```

Comment:
Can you mention Navigator.requestFocus here and explain how they relate?

Reply:
I made it default to `Navigator.requestFocus`, so it avoids needing two values to control whether to request focus.